plans/sep8.md

```diff
@@ -23,7 +23,7 @@ The high-level flow is:
     - Display the message if included in the response
   - if action_required
     - Display the message
-    - Redirect to action_url
+    - Redirect to action_url based on action_method and action_fields
```

Comment:
➕ There's more to it than a simple redirect.

If the action_method is `GET` you can load the `action_url` in browser or app.

If the action_method is `POST` then the app must make a request to the POST endpoint first, and then depending on the result either stops there, or then continues on to loading the `next_url` in browser or app.

See [this section](http://stellar.org/protocol/sep-8#action-url).

Reply:
https://github.com/stellar/js-stellar-wallets/pull/201/commits/86fcdf3bb6c2d89be30c434849b506bde91741e9